ccd-definition/CaseType.json

```diff
@@ -9,9 +9,9 @@
   },
   {
     "LiveFrom": "01/01/2017",
-    "ID": "SHARED_STORAGE",
-    "Name": "Shared storage",
-    "Description": "Shared storage",
+    "ID": "Shared_Storage_DRAFTType",
+    "Name": "Shared storage draft",
+    "Description": "Shared storage draft",
```

Comment:
To be honest I wouldn't but "draft" in here, this is displayed to the user I don't think we want to show "draft" for them. I would like it possibly as shared storage.